packages/lib/src/components/Dropin/components/DropinComponent.tsx

```diff
@@ -92,6 +92,25 @@ export class DropinComponent extends Component<DropinComponentProps, DropinCompo
         this.setState({ activePaymentMethod: null });
     }
 
+    private onOrderCancel = data => {
```

Comment:
Can the `data` type and return type of the function be defined here?

Reply:
I typed the return functions, I can also add types to this one, but they where being inferred correctly